engine/deeplearning/src/main/java/org/opennms/alec/engine/deeplearning/ClasspathUtils.java

```diff
@@ -50,8 +52,14 @@
  * @author stack overflow
  */
 class ClasspathUtils {
+    public static final int THRESHOLD_ENTRIES = 10000;// 1 GB
```

Comment:
I think this comment is on the wrong variable

Reply:
also fixed